experiments/functional/cvr-scaleup/test.yml

```diff
@@ -225,6 +238,26 @@
           delay: 10
           retries: 50
 
+        - block: 
+
+            - name: Getting the taget name 
+              shell:  kubectl get pod -n {{ operator_ns }} -l openebs.io/persistent-volume={{ pv.stdout }} -o jsonpath='{.items[0].metadata.name}' 
+              register: target_name
+
+            - name: Checking for reconstructing state of newely created cvr 
+              shell: kubectl get cvr {{ pv.stdout }}-{{ csp_name.stdout_lines[0] }} -n {{ operator_ns }} -o jsonpath='{.status.phase}'
```

Comment:
we can't assure that the new cvr will be created on csp with index 0. Let us find the csp name prior and use it @shashank855